companion-generator/src/main/java/com/gradle/cucumber/companion/generator/CompanionGenerator.java

```diff
@@ -40,25 +42,25 @@ public static void create(CompanionFile companionFile) throws IOException {
                 bw.write("package ");
                 bw.write(companionFile.getPackageName().get());
                 bw.write(";");
-                bw.newLine();
-                bw.newLine();
+                bw.write(NEW_LINE);
+                bw.write(NEW_LINE);
             }
             bw.write("@org.junit.platform.suite.api.Suite");
-            bw.newLine();
+            bw.write(NEW_LINE);
             bw.write("@org.junit.platform.suite.api.SelectClasspathResource(\"");
-            bw.write(companionFile.getClassPathResource());
+            bw.write(companionFile.getClassPathResource().replace('\\', '/'));
```

Comment:
❔ I don't know how `SelectClasspathResource` works. Doesn't it expect a proper path that it can access? Or is it just an informative annotation?

Reply:
It needs to be a path on the classpath, not in the file system. In either case, using `/` instead of `\` should work fine.

Reply:
Understood, thank you.